docs/api-reference/webgl/copy-and-blit.md

```diff
@@ -60,50 +58,33 @@ This function makes calls to the following WebGL APIs:
 [`gl.readPixels`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/readPixels), [`gl.bindFramebuffer`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/bindFramebuffer), [`gl.bindBuffer`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/bindBuffer)
 
 
-## Copy Functions
-
-### copyToTexture(opts: Object) : Texture
+### readPixelsToDataUrl(source : Framebuffer|Texture, opts: Object) : Data URL
```

Comment:
Not sure about the `readPixels` vs `copy` here as the result is an encoded image, i.e. the app won't be able to get to the pixels. I think the format can be specified (e.g. it doesn't have to be PNG).